File: output/StarCoder/avatar/Java/Python/atcoder_ABC164_A.py
import sys

scList = []

while True:
    try:
        scList.append(input())
    except:
        break

numlist = scList[0].split()

Sheep = int(numlist[0])
Wolve = int(numlist[1])

if Wolve >= Sheep:
    print("unsafe")
else:
    print("safe")

File: output/ChatGPT/avatar/Java/C++/atcoder_ABC164_A.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    vector<string> scList;
    string line;
    while (getline(cin, line)) {
        scList.push_back(line);
    }
    
    string numlist = scList[0];
    int pos = numlist.find(' ');
    int Sheep = stoi(numlist.substr(0, pos));
    int Wolve = stoi(numlist.substr(pos + 1));
    
    if (Wolve >= Sheep) {
        cout << "unsafe" << endl;
    } else {
        cout << "safe" << endl;
    }
    
    return 0;
}